docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -93,3 +93,35 @@ Workaround: Delete the existing {project} VM and start again.
 $ minishift delete --force
 $ minishift start
 ----
+
+[[Remove-password-from-keychain]]
+== Removing subscription password from OS native keychain
+
+This only applies if you are using a subscription capable distro like RHEL with {project}. 
+
+=== Windows
+
+On Windows, the password is stored using *Credential Manager*.
```

Comment:
s/the password/by default the password/ . Also it is same for all Windows versions? e.g. between Windows 7 and Windows 10.